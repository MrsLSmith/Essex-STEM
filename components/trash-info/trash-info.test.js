import "react-native";
import React from "react";
import TrashInfo from "./index";
import renderer from "react-test-renderer";

it("renders correctly", () => {
    const tree = renderer.create(<TrashInfo/>).toJSON();
    expect(tree).toMatchSnapshot();
});
