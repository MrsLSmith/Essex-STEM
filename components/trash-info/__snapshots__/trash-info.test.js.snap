// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "white",
      "margin": 10,
      "padding": 20,
    }
  }
>
  <Text
    style={
      Object {
        "fontSize": 20,
      }
    }
  >
    Each town handles trash bags differently. Find the rules for your town
  </Text>
  <Text
    style={
      Object {
        "fontSize": 16,
        "marginTop": 20,
      }
    }
  >
    Trash bag tracking starts Thursday 04/30/2020 and ends Tuesday 05/05/2020
  </Text>
</View>
`;
